.github/workflows/eslint_disable_check.py

```
@@ -58,14 +58,19 @@
     """
     eslint_found = False
 
+    # Walk through the 'src' directory and check only .ts and .tsx files
     for root, dirs, files in os.walk(os.path.join(directory, 'src')):
```

Ruff (0.8.0)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)